main.go

```diff
@@ -164,55 +179,88 @@ func mainE(config Configs) error {
 	}
 
 	fmt.Println()
+	log.Infof("Export Artifacts:")
 
-	log.Infof("Export APKs:")
-	fmt.Println()
-
-	apks, err := getArtifacts(gradleProject, started, config.APKPathPattern, false)
+	artifacts, err := getArtifacts(gradleProject, started, appPatterns, false)
 	if err != nil {
-		return fmt.Errorf("failed to find apks, error: %v", err)
+		return fmt.Errorf("failed to find artifacts, error: %v", err)
+	}
+
+	var artNames []string
+	for _, a := range artifacts {
+		artNames = append(artNames, a.Name)
 	}
 
-	if len(apks) == 0 {
-		log.Warnf("No apks found with pattern: %s", config.APKPathPattern)
-		log.Warnf("If you have changed default APK export path in your gradle files then you might need to change APKPathPattern accordingly.")
+	log.Donef("Found artifacts by provided pattern:")
+
+	// Filter artifacts by build type
+	var filteredArtifacts []gradle.Artifact
+	for _, a := range artifacts {
+		if filepath.Ext(a.Path) == "."+config.BuildType {
```

Comment:
what if the user wants to export other than the build type? he has the rights to set export filters so we need to export everything that is matching for the pattern (for example code coverage files or reports etc), the only thing that has changed is that .apk and .aab will be exported to a different env, no need to print dummy log about the extension as it is visible from the file name also. The current log doesn't really contains useful information that will answer the users question if they don't find someting,